src/main/java/com/google/step/servlets/CommentPostServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.step.servlets;
+
+import com.google.step.datamanager.CommentManager;
+import com.google.step.datamanager.CommentManagerDatastore;
+import com.google.step.model.Comment;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/comments/")
+public class CommentPostServlet extends HttpServlet {
+
+  private CommentManager manager = new CommentManagerDatastore();
+
+  public CommentPostServlet(CommentManager commentManager) {
```

Comment:
missing constructor without arguments. I believe the appengine won't work without it.